src/resource.spec.js

```diff
@@ -224,12 +224,24 @@ describe('Resource', function () {
     expect(mockContext.httpGet).toHaveBeenCalledWith(resource);
   });
 
+  it('loads a resource if it has been synced but stale', function () {
+    resource.$syncTime = new Date('2016-01-01').getTime();
+    resource.$load(Date.now());
+    expect(mockContext.httpGet).toHaveBeenCalledWith(resource);
+  });
+
   it('does not load a resource again if it has been synced', function () {
     resource.$syncTime = Date.now();
     resource.$load();
     expect(mockContext.httpGet).not.toHaveBeenCalled();
   });
 
+  it('does not load a resource again if it has been synced and not stale', function () {
+    resource.$syncTime = Date.now();
+    resource.$load(new Date('2016-01-01').getTime());
```

Comment:
These tests assume `Date.now()` is after 2016-01-01. Not unreasonable, but why not just use something like `new Date('2016-01-02')`?